### PR TITLE
Added StringContains() and StringContainsChar()

### PR DESCRIPTION
**Pull request by olehermanse**
Inlined functions for better readability.

### CODE REVIEW

File: libutils/string_lib.h [hunk 1]
```diff
@@ -102,6 +102,16 @@ static inline bool CharIsPrintableAscii(const char c)
 }
 bool EmptyString(const char *s);
 
+static inline bool StringContains(const char *const a, const char *const b)
```
Comment (vpodzime) [inline libutils/string_lib.h:105]: `needle` and `hay` would be better names for the parameters.

**Review by vpodzime — COMMENTED** [review]
Looks good otherwise.

File: libutils/string_lib.h (hunk 1 above)
Comment (olehermanse) [reply libutils/string_lib.h]: Done.

**Review by vpodzime — APPROVED** [review]
ACK

**Pull request merged**